health/statuscheck/plugin_impl_statuscheck.go

```diff
@@ -195,6 +217,36 @@ func (p *Plugin) ReportStateChange(pluginName core.PluginName, state PluginState
 	}
 }
 
+func (p *Plugin) reportInterfaceStateChange(data *status.InterfaceStatus_Interfaces) {
+	// Filter interfaces without internal name
```

Comment:
This doesn't synchronize access with `p.access` and might cause race condition on multiple calls to `ReportStateChangeWithMeta`.